lib/OpenQA/ServerStartup.pm

```diff
@@ -62,6 +62,11 @@ sub read_config {
         audit => {
             blacklist => '',
         },
+        amqp => {
```

Comment:
I don't think we want a default for a username and password - possibly for exchange and topic_prefix, but even that sounds very specific to the instance.